contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -89,6 +101,89 @@ public VoldemortSwapJob(String id,
         this.failedFetchStrategyList = failedFetchStrategyList;
         this.clusterName = clusterName;
         this.buildPrimaryReplicasOnly = buildPrimaryReplicasOnly;
+        this.props = props;
+    }
+
+    private void runDistcp(Path from, Path to) throws Exception {
+        info("     source: " + from);
+        info("destination: " + to);
+        EmbeddedGobblin embeddedGobblin = new EmbeddedGobblinDistcp(from, to).mrMode();
+
+        // Used for global throttling"
+        embeddedGobblin.distributeJar("lib/*");
+
+        for (Map.Entry<String, String> entry : this.props.entrySet()) {
+            if (entry.getKey() instanceof String && ((String) entry.getKey()).startsWith("distcpConf.")) {
+                String key = ((String) entry.getKey()).substring("distcpConf.".length());
+                embeddedGobblin.setConfiguration(key, entry.getValue().toString());
+            }
+        }
+        JobExecutionResult result =  embeddedGobblin.run();
```

Comment:
Which user will be used to run the distcp job?
I didn't see the related config here.